src/components/dashboard/main-content/course-enrollments/data/utils.js

```diff
@@ -31,7 +31,10 @@ export const transformCourseEnrollment = (rawCourseEnrollment) => {
   // present if the learner has made progress. If the learner has not made progress,
   // we should link to the main course run URL. Similarly, if the resume course link
   // is not set in the API response, we should fallback on the normal course link.
-  courseEnrollment.linkToCourse = courseEnrollment.resumeCourseRunUrl || courseEnrollment.courseRunUrl;
+  // courseEnrollment.linkToCourse = courseEnrollment.resumeCourseRunUrl || courseEnrollment.courseRunUrl;
+  // TODO: uncomment out the previous line to allow for unique resume url
+  // https://2u-internal.atlassian.net/browse/ENT-8065
+  courseEnrollment.linkToCourse = courseEnrollment.courseRunUrl;
```

Comment:
@kiram15 I'm not sure this is entirely the correct fix as it removes the functionality that satisfies the product/UX requirement that the "Resume" URL should bring the learner to the correct xblock/unit in the course.

The issue isn't that the `courseEnrollment.resumeCourseRunUrl` was getting used; the issue is that its value is a relative URL instead of absolute. For example, the `courseRunUrl` includes `https://learning.edx.org` in the URL whereas `resumeCourseRunUrl` does not (i.e., `"resume_course_run_url": "/courses/course-v1:BerkeleyX+CS198.2x+3T2019/jump_to/block-v1:BerkeleyX+CS198.2x+3T2019+type@html+block@349076b9f18947c183a7044892ee2047",`).

The fix to keep the functionality with the "Resume" CTA working as expected, we'd want to ensure the `resumeCourseRunUrl` includes the `https://courses.edx.org` prefix. Ideally, this would be returned by the `enterprise-course-enrollments` API instead of returning the relative URL.

It appears this is more of an API bug than a frontend bug.